src/components/shared/seo/seo.jsx

```diff
@@ -26,38 +17,29 @@ const SEO = ({ data: { title, description, image, slug } = {}, facebook } = {})
           siteUrl
           siteImage
           siteLanguage
-          authorTwitterAccount
         }
       }
     }
   `);
 
-  const currentTitle = title || siteTitle;
-  const currentDescription = description || siteDescription;
-  const currentUrl = slug ? `${siteUrl}/${slug}` : siteUrl;
-  const currentImage = image || siteImage;
-  const currentImagePath = createMetaImagePath(currentImage, siteUrl);
```

Comment:
This change restricts usage of SEO components as we can no longer pass custom meta params and rely only on what is defined in gatsby-config.

Reply:
Yeah we'll need to work on it later